tendrl/node_agent/collectd/collectors/ceph/tendrl_ceph_cluster_iops.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
```

Comment:
Add "monitoring" before collectd/... is what I am asking. Not asking to merge gluster and ceph